HandEyeCalibration/Transform/Transform.cpp

```diff
@@ -0,0 +1,330 @@
+/*
+Convenience functions for pose and co-ordinate system transformations.
+*/
+
+#include <Eigen/Core>
+#include <Eigen/Dense>
+#include <Eigen/Geometry>
+
+#include <opencv2/core/core.hpp>
+
+#include <cmath>
+#include <iostream>
+
+#define _USE_MATH_DEFINES
+
+namespace TransformUtils
+{
+    enum Rotation
+    {
+        XYZ_INTRINSIC,
+        XYZ_EXTRINSIC,
+        ZYX_INTRINSIC,
+        ZYX_EXTRINSIC
+    };
+
+    struct Rotation_s
+    {
+        Rotation rot;
+        std::string short_desc;
+        std::string rpy_name;
+    };
+
+    cv::Mat eigen2cv(const Eigen::MatrixXd &eigenMat)
```

Comment:
Camel case for consistency

Christian is useing cvToEigen so I suggest going for eigenToCv, check:

https://github.com/zivid/cpp-extra-samples/blob/master/HandEyeCalibration/UtilizeEyeInHandCalibration/UtilizeEyeInHandCalibration.cpp